### PR TITLE
Wiki Guide Integration

### PR DESCRIPTION
**Pull request by Misttyy**
## About The Pull Request

This PR adds new defines for every marine role I could find on the wiki that leads to them -- Expanding from this, every entry text for every marine role has had a wiki link to its respective page added. 
## Why It's Good For The Game

Our wiki maintainer and contributors work hard. This should make it far easier for the average player to get to see their work and make a more enjoyable experience overall.

## Changelog

:cl: Mistfrag
qol: Linked respective wiki page to all Marine roles.
code: Added defines for every wiki page on marines.
/:cl:

### CODE REVIEW

**Comment by Misttyy** [comment]
Got the errors ironed out. Could use help from a more experienced dev on adding a Xeno equivalent of entry_message for Larva and Queen since the var doesn't work.

**Comment by fira** [comment]
Please don't use ////// as separators, three or more slashes are interpreted as DMDOC annotations and will be used as documentation for the define below

**Comment by fira** [comment]
For Queen, you could use /datum/job/antag/xenos/queen/announce_entry_message

**Comment by Misttyy** [comment]
Removed the xeno bits of this. I'm going to need to do some more codework than I thought to get them working and I'd rather not stall this PR for it.

**Review by Geevies — CHANGES_REQUESTED** [review]
echoing what fira said about ///

File: code/modules/mob/living/carbon/xenomorph/xeno_defines.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	var/tier = 0
 	var/dead_icon = "Drone Dead"
 	var/language = LANGUAGE_XENOMORPH
-
+	var/entry_message_intro
```
Comment (Geevies) [inline code/modules/mob/living/carbon/xenomorph/xeno_defines.dm:9]: remove this stub

File: code/modules/mob/living/carbon/xenomorph/castes/Queen.dm [hunk 2]
```diff
@@ -23,8 +23,7 @@
 	is_intelligent = 1
 	evolution_allowed = FALSE
 	fire_immunity = FIRE_IMMUNITY_NO_DAMAGE|FIRE_IMMUNITY_NO_IGNITE
-	caste_desc = "The biggest and baddest xeno. The Queen controls the hive and plants eggs"
-	spit_types = list(/datum/ammo/xeno/toxin/queen, /datum/ammo/xeno/acid/medium)
```
Comment (Geevies) [inline code/modules/mob/living/carbon/xenomorph/castes/Queen.dm:27]: readd spit types

**Review by Geevies — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/Queen.dm (hunk 2 above)
Comment (Misttyy) [reply code/modules/mob/living/carbon/xenomorph/castes/Queen.dm:27]: I'm retarded

File: code/__DEFINES/urls.dm [hunk 3]
```diff
@@ -1,30 +1,64 @@
-// Changelog
+/// Changelog ///
```
Comment (Geevies) [inline code/__DEFINES/urls.dm:1]: ///

**Review by Geevies — CHANGES_REQUESTED** [review]

**Comment by Misttyy** [comment]
should be good

**Comment by Misttyy** [comment]
I'm going to fix the /// when I get home. 

I need better glasses.

**Comment by Misttyy** [comment]
@fira @Geevies actually fixed now

**Pull request merged**